packages/vscode-lit-plugin/package.json

```diff
@@ -116,26 +117,59 @@
 			],
 			"command": "rm -rf ../../../packaged-extension/ && mkdir ../../../packaged-extension/ && unzip -qq ./out/packaged.vsix -d ../../../packaged-extension/",
 			"clean": false
+		},
+		"bundle": {
+			"dependencies": [
+				"../lit-analyzer:build"
+			],
+			"command": "node esbuild.script.js",
+			"files": [
+				"src/**/*",
+				"esbuild.script.js"
+			],
+			"output": [
+				"built/bundle.js"
+			]
+		},
+		"make-built-dir": {
+			"dependencies": [
+				"../ts-lit-plugin:build",
+				"bundle"
+			],
+			"files": [
+				"../ts-lit-plugin/package.json",
+				"./package.json",
```

Comment:
Very nit, but there's a mix of globs with and without the `./` prefix, which is a no-op. Personally I think I find it easier to read without the prefix (so just `package.json` instead of `./package.json`), though main comment is just that it'd be nice to be consistent.